src/components/navigation/footer-navigation/footer-navigation.js

```diff
@@ -97,7 +97,12 @@ export default class Navigation extends React.PureComponent {
           />
         </span>
       ) : link.external ? (
-        <ExternalLink link={link} key={i} />
+        <ExternalLink
+          link={link}
+          key={i}
+          target="_blank"
```

Comment:
This should be inside the external-link-box.js

Reply:
Oh! You are right!

Reply:
It should go to the link, external link box is not used here. I'll fix both.